buffer/src/main/java/io/netty/buffer/ByteBufUtil.java

```diff
@@ -437,18 +445,27 @@ private static int writeUtf8(AbstractByteBuf buffer, CharSequence seq, int len)
     }
 
     /**
+     * Encode a {@link CharSequence} in <a href="http://en.wikipedia.org/wiki/ASCII">ASCII</a> and write
+     * it to a {@link ByteBuf} allocated with {@code alloc}.
+     * @param alloc The allocator used to allocate a new {@link ByteBuf}.
+     * @param seq The characters to write into a buffer.
+     * @return The {@link ByteBuf} which contains the <a href="http://en.wikipedia.org/wiki/ASCII">ASCII</a> encoded
+     * result.
+     */
+    public static ByteBuf writeAscii(ByteBufAllocator alloc, CharSequence seq) {
+        // ASCII uses 1 byte per char
+        ByteBuf buf = alloc.buffer(seq.length());
```

Comment:
`writeAscii` should not throw to be consistent with `getBytes()`. If it throws isn't that an error?